musica/mechanism_configuration/branched.py

```diff
@@ -60,53 +60,233 @@ def __init__(
+                species = Species(name=rc.species_name)
+                result.append((rc.coefficient, species))
+            else:
+                # Just the species
+                species = Species(name=rc.species_name)
+                result.append(species)
+        return result
+
+    @alkoxy_products.setter
+    def alkoxy_products(self, value: List[Union[Species, Tuple[float, Species]]]):
+        """Set the alkoxy products, converting from Python to C++ objects."""
+        cpp_products = []
+        for p in value:
+            if isinstance(p, Species):
+                cpp_products.append(_ReactionComponent(p.name))
+            elif isinstance(p, tuple) and len(p) == 2:
+                coefficient, species = p
+                cpp_products.append(_ReactionComponent(species.name, coefficient))
+            else:
+                raise ValueError(f"Invalid alkoxy product format: {p}")
+        self._instance.alkoxy_products = cpp_products
+
+    @property
+    def gas_phase(self) -> str:
+        """Get the gas phase name."""
+        return self._instance.gas_phase
+
+    @gas_phase.setter
+    def gas_phase(self, value: Union[Phase, str]):
+        """Set the gas phase."""
+        if isinstance(value, Phase):
+            self._instance.gas_phase = value.name
+        elif isinstance(value, str):
+            self._instance.gas_phase = value
+        else:
+            raise ValueError(f"Invalid gas_phase type: {type(value)}")
+
+    @property
+    def other_properties(self) -> Dict[str, Any]:
+        """Get the other properties."""
+        return self._instance.other_properties
+
+    @other_properties.setter
+    def other_properties(self, value: Dict[str, Any]):
+        """Set the other properties."""
+        self._instance.other_properties = value
+
+    def serialize(self) -> Dict:
+        """
+        Serialize the Branched object to a dictionary using only Python-visible data.
+
+        Returns:
+            Dict: A dictionary representation of the Branched object.
+        """
+        serialize_dict = {
+            "type": "BRANCHED_NO_RO2",
+            "name": self.name,
+            "X": self.X,
+            "Y": self.Y,
+            "a0": self.a0,
+            "n": self.n,
+            "reactants": ReactionComponentSerializer.serialize_list_reaction_components(
+                self._instance.reactants),
+            "nitrate products": ReactionComponentSerializer.serialize_list_reaction_components(
+                self._instance.nitrate_products),
+            "alkoxy products": ReactionComponentSerializer.serialize_list_reaction_components(
+                self._instance.alkoxy_products),
+            "gas phase": self.gas_phase,
+        }
+        _add_other_properties(serialize_dict, self.other_properties)
+        return _remove_empty_keys(serialize_dict)
 
     @staticmethod
-    def serialize(instance) -> Dict:
+    def serialize_static(instance) -> Dict:
```

Comment:
Create a local tempoerary varialbe that is of class Branched andthen call `serialize` so that the dictionary is not duplicated